packages/template-retail-react-app/config/default.js

```diff
@@ -60,7 +60,8 @@ module.exports = {
         dataCloudAPI: {
             appSourceId: '7ae070a6-f4ec-4def-a383-d9cacc3f20a1',
             tenantId: 'g82wgnrvm-ywk9dggrrw8mtggy.pc-rnd'
-        }
+        },
+        partialHydrationEnabled: false
```

Comment:
As per Ben's suggestion, we now toggle the partial hydration feature via the config file (not the constants file).